php-checks/src/main/java/org/sonar/php/checks/PHPDeprecatedFunctionUsageCheck.java

```diff
@@ -179,4 +201,14 @@ private void checkDefineArguments(FunctionCallTree tree) {
     }
   }
 
+  private void checkStreamFilterAppendArguments(FunctionCallTree tree) {
+    SeparatedList<ExpressionTree> arguments = tree.arguments();
+    if (arguments.size() >= 2) {
+      ExpressionTree filterNameArgument = arguments.get(1);
+      if (filterNameArgument.is(Kind.REGULAR_STRING_LITERAL) && CheckUtils.trimQuotes((LiteralTree) filterNameArgument).equals("string.strip_tags")) {
+        context().newIssue(this, filterNameArgument, "Remove this highly discouraged 'string.strip_tags' filter usage.");
```

Comment:
Use double quote like for others, and maybe make the message consistent with others as well: 
`Remove this deprecated 'string.strip_tags' filter usage.`

No need to say that it's highly discouraged - we are not explaining why in the rule - having it deprecated is enough.